mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -154,31 +156,56 @@ class LinearSVC @Since("2.2.0") (
   def setAggregationDepth(value: Int): this.type = set(aggregationDepth, value)
   setDefault(aggregationDepth -> 2)
 
+  /**
+   * Set block size for stacking input data in matrices.
```

Comment:
We might provide a little more comment about what this does. Increasing it increases performance, but at the risk of what, slowing down on sparse input?

Reply:
The choice of size needs tuning, it depends on dataset sparsity and numFeatures, Increasing it may not always increases performance.